src/gui/FileBrowser.cpp

```diff
@@ -656,14 +663,22 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent * e )
 		patternEditorHeader->setDisabled(true);
 		contextMenu.addAction(patternEditorHeader);
 		contextMenu.addActions( getContextActions(file, false) );
-
-		// We should only show the menu if it contains items
-		if (!contextMenu.isEmpty()) { contextMenu.exec( e->globalPos() ); }
+		break;
+	}
+	case TypeDirectoryItem: {
+		auto dir = dynamic_cast<Directory*>(item);
+		QString dirname = dir->fullName();
+		contextMenu.addAction(QIcon(embed::getIconPixmap("folder")), tr("Open in %1").arg(fileManager), [=, this] {
```

Comment:
```suggestion
		contextMenu.addAction(QIcon(embed::getIconPixmap("folder")), tr("Open in %1").arg(fileManager), [=] {
```

Reply:
Clang (macOS) will error (`-Werror`) if they're left for areas that they're unused.   The only unused `this` that must be removed are the ones that no longer call a FileBrowser function now, which are the calls to `openDir` and `reveal`.